docs/semgrep-ci/configuration-reference.md

```diff
@@ -118,4 +118,25 @@ To change the job timeout from the default of 1800 seconds. Set to 0 to disable
 SEMGREP_TIMEOUT="300"
 ```
 
+## Enabling GitLab MR comments (non-standard CI configuration)
+
+The configuration provided in this section is not needed for a standard Semgrep in CI setup. Use this configuration only when you are using GitLab runners to provide MR comments while you are not using GitLab MR job. In the code snippet below, magenta-colored values are placeholders that you can substitute. Set up the following environment variables within your command line to allow Semgrep to create MR comments in GitLab:
+
+<pre class="language-bash"><code>
+export GITLAB_CI='true'<br/>
+export CI_PROJECT_PATH='<span className="placeholder">USERNAME</span>/<span className="placeholder">PROJECTNAME</span>'<br/>
+export CI_MERGE_REQUEST_PROJECT_URL='https://gitlab.com/<span className="placeholder">USERNAME</span>/<span className="placeholder">PROJECTNAME</span>'<br/>
+export CI_PROJECT_URL="$CI_MERGE_REQUEST_PROJECT_URL"<br/>
+export CI_COMMIT_SHA='<span className="placeholder">COMMIT-SHA-VALUE</span>'<br/>
+export CI_COMMIT_REF_NAME='<span className="placeholder">REF</span>'<br/>
+export CI_MERGE_REQUEST_TARGET_BRANCH_NAME='<span className="placeholder">BRANCH_NAME</span>'<br/>
+export CI_JOB_URL='<span className="placeholder">JOB_URL</span>'<br/>
+export CI_PIPELINE_SOURCE='merge_request_event'<br/>
+export CI_MERGE_REQUEST_IID='<span className="placeholder">REQUEST_IID</span>'<br/>
+export CI_MERGE_REQUEST_DIFF_BASE_SHA='<span className="placeholder">SHA</span>'<br/>
+export CI_MERGE_REQUEST_TITLE='<span className="placeholder">MERGE_REQUEST_TITLE</span>'<br/>
+</code></pre>
+
+Replace magenta-colored placeholders in the code snippet above with your specific values. For more information on all of these variables see GitLab documentation [Predefined variables reference](https://docs.gitlab.com/ee/ci/variables/predefined_variables.html). You can find an exhaustive example with sample values in [List all environment variables](https://docs.gitlab.com/ee/ci/variables/index.html#list-all-environment-variables).
```

Comment:
```suggestion
Replace <span className="placeholder">magenta-colored placeholders</span> in the code snippet above with your specific values. For more information on all of these variables see GitLab documentation [Predefined variables reference](https://docs.gitlab.com/ee/ci/variables/predefined_variables.html). You can find an exhaustive example with sample values in [List all environment variables](https://docs.gitlab.com/ee/ci/variables/index.html#list-all-environment-variables).
```

Reply:
@nbrahms this is a good idea, but might be still confusing because of the lower case letters used. What about:

```suggestion
Replace magenta-colored placeholders in the code snippet above with your specific values (for example <code><span className="placeholder">USERNAME</span></code>). For more information on all of these variables see GitLab documentation [Predefined variables reference](https://docs.gitlab.com/ee/ci/variables/predefined_variables.html). You can find an exhaustive example with sample values in [List all environment variables](https://docs.gitlab.com/ee/ci/variables/index.html#list-all-environment-variables).
```